e2e/tests.spec.ts

```diff
@@ -0,0 +1,42 @@
+import {test, expect} from '@playwright/test'
+import {writeInMonaco} from './fixtures'
+
+test('has title "Zod Playground', async ({page}) => {
+  await page.goto('/')
+
+  await expect(page).toHaveTitle(/Zod Playground/)
+})
+
+test('has header with title, share, theme toggler and github repo link', async ({
+  page,
+}) => {
+  await page.goto('/')
+
+  await expect(page.getByText('Zod Playground')).toBeVisible()
+  await expect(page.getByRole('button', {name: 'Share'})).toBeVisible()
+  await expect(page.getByLabel('Toggle color scheme')).toBeVisible()
+  await expect(page.getByRole('banner').getByRole('link')).toHaveAttribute(
+    'href',
+    'https://github.com/marilari88/zod-playground',
+  )
+})
+
+test('zod version switch', async ({page}) => {
+  await page.goto('/')
+
+  await page.getByRole('button', {name: 'v3.23.8'}).click()
+  await page.getByRole('option', {name: '3.23.7'}).click()
+
+  await expect(page.getByRole('button', {name: 'v3.23.8'})).not.toBeVisible()
+  await expect(page.getByRole('button', {name: 'v3.23.7'})).toBeVisible()
```

Comment:
maybe we should mock versions

Reply:
Don't know, these are e2e, if some versions change it's right for the tests to break

Reply:
in fact my concern is related to the change of the latest version that should take place at each release of Zod

Reply:
https://github.com/marilari88/zod-playground/pull/55#discussion_r1710453191